downloads/install.sh

```diff
@@ -89,10 +89,6 @@ esac
 case $OSTYPE in
 "linux-gnu"*)
     OS="linux"
-    if [[ $ARC == "aarch64" ]]; then
```

Comment:
nit: `$ARCH` I think is the common variable name for the architecture.

Reply:
Why did you comment on the code being deleted?